packages/liquidator/index.js

```diff
@@ -155,6 +155,57 @@ async function run({
       }
     };
 
+    // Checks whether the liquidator bot wallet balance is >= total open positions * minimum sponsor amount
+    const calculateBotWalletBalance = async () => {
+      try {
+        // update client state
+        this.client = financialContractClient;
+        this.web3 = this.client.web3;
+        this.syntheticToken = syntheticToken;
+
+        await this.client.update();
```

Comment:
we don't really need to update `financialContractClient` here again. Since, liquidator.update updates the financialContractClient too.